pallets/attesters/contracts/contracts/attestationsVerifier.sol

```diff
@@ -40,26 +40,27 @@ contract AttestationsVerifier {
     }
 
     function receiveAttestationBatch(
-        address[] memory newCommittee,
+        address[] memory nextCommittee,
         address[] memory bannedCommittee,
-        bytes32[] memory confirmedSFXs,
-        bytes32[] memory revertedSFXs,
+        bytes32[] memory committedSfx,
+        bytes32[] memory revertedSfx,
         uint32 index,
         bytes32 expectedBatchHash,
         bytes[] memory signatures
     ) public {
-        Batch memory batch = Batch(newCommittee, bannedCommittee, confirmedSFXs, revertedSFXs, index);
-        bytes32 batchMessageHash = keccak256(batchEncodePacked(batch));
-        require(batchMessageHash == expectedBatchHash, "Batch hash mismatch");
+        Batch memory batch = Batch(nextCommittee, bannedCommittee, committedSfx, revertedSfx, index);
+        // TODO: 
+        // bytes32 batchMessageHash = keccak256(batchEncodePacked(batch));
+        // require(batchMessageHash == expectedBatchHash, "Batch hash mismatch");
 
-        require(verifySignedByActiveCommittee(batchMessageHash, signatures), "Signatures verification failed");
+        require(verifySignedByActiveCommittee(expectedBatchHash, signatures), "Signatures verification failed");
 
-        require(batch.index == currentBatchIndex + 1, "Batch index mismatch");
+        require(batch.index == currentBatchIndex, "Batch index mismatch");
```

Comment:
we should keep comparison to batch + 1, but we'd need to add start index to constructor params:

>  constructor(address[] memory initialCommittee, uint32 startIndex)

Reply:
hmmm but we want to start from batch 0, then with this we should pass to contructor index = -1 to process batch 0.